default.json

```diff
@@ -80,6 +80,17 @@
       "depNameTemplate": "mikefarah/yq",
       "datasourceTemplate": "github-releases"
     },
+    {
+      "fileMatch": [
+        "(^|/|\\.)Dockerfile$",
+        "(^|/)Dockerfile[^/]*$"
+      ],
+      "matchStrings": [
+        "ENV CONTROLLER_GEN_VERSION(\\=|\\s)(?<currentValue>.*?)\\n"
```

Comment:
We could add an alternative pattern for: `CONTROLLER_GEN_VER := ...` as per Makefile examples above.

Reply:
We'd need to first change the name of the ENV in the respective Dockerfile, or could we just make the pattern shorter here?

Reply:
Either way really, we can either change in the repositories to use a single pattern, or add both patterns here until that take place. I can see 2 patterns across 4 files:

![image](https://github.com/rancher/renovate-config/assets/5452977/d62d8473-7d94-49c1-83a6-7a8a6b6b168f)